test/workflows/components/workflows.libsonnet

```diff
@@ -337,16 +351,24 @@
                 ],
               },
             },  // checkout
-            $.parts(namespace, name, overrides).e2e(prow_env, bucket).buildTemplate("build", [
-              "python",
-              "-m",
-              "kubeflow.tf_operator.release",
-              "build",
-              "--src_dir=" + srcDir,
-              "--registry=" + params.registry,
-              "--project=" + project,
-              "--version_tag=" + versionTag,
-            ]),  // build
+            $.parts(namespace, name, overrides).e2e(prow_env, bucket).buildTemplate("build", "gcr.io/kaniko-project/executor:v1.0.0", [
+              #"scripts/build.sh",
+              "/kaniko/executor",
+              "--dockerfile=" + srcDir + "/Dockerfile",
+              "--context=dir://" + srcDir,
+              "--destination=" + "527798164940.dkr.ecr.us-west-2.amazonaws.com/tf-operator:$(PULL_BASE_SHA)",
```

Comment:
So one ECR registry we need to create is:

`527798164940.dkr.ecr.us-west-2.amazonaws.com/tf-operator`

Reply:
@PatrickXYS Yes, I believe that is enough for tf-operator

Reply:
Hi @PatrickXYS, is the registry created ?

Reply:
Yes it has been created. Before we move foward, I've already DM @Bobgy for webhook configuration. Will keep you updated @ChanYiLin